modules/entity/src/models.ts

```diff
@@ -40,18 +38,11 @@ export type UpdateNum<T> = {
 
 export type Update<T> = UpdateStr<T> | UpdateNum<T>;
 
-export interface EntityStateStr<T> {
-  ids: string[];
+export interface EntityState<T> {
+  ids: any[];
```

Comment:
If the possible values of `ids` are either string or number, it better to be explicit about that `ids: string[] | number[];`.

Reply:
This won't work with TypeScript